upup/pkg/fi/cloudup/urls.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cloudup
+
+import (
+	"github.com/golang/glog"
+	"k8s.io/kops"
+	"os"
+	"strings"
+)
+
+// baseUrl caches the BaseUrl value
+var baseUrl string
+
+// BaseUrl returns the base url for the distribution of kops - in particular for nodeup & docker images
+func BaseUrl() string {
+	if baseUrl != "" {
+		// Avoid repeated logging
+		return baseUrl
+	}
+
+	// We prefer KOPS_BASE_URL, but we will accept KOPS_URL (for now!)
+	baseUrl = os.Getenv("KOPS_BASE_URL")
+	if baseUrl == "" {
+		baseUrl = os.Getenv("KOPS_URL")
```

Comment:
Oh, sorry, if I had read the whole PR it would've been obvious. Yeah, file a bug and I'll push this through gradually.